### PR TITLE
Us3109 get allowed plans

### PR DESCRIPTION
**Pull request by karansharmaufl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,40 @@
-# GetAllowedPlans
-This service in SellerDigital gets all the allowed plans against a given loan.
+## GetStipulations
```
Comment (akshayjoshi-hws) [inline README.md]: fix readme with proper title

File: GetAllowedPlans.template [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+    "GetAllowedPlansStarterLambda": {
+      "Type": "AWS::Lambda::Function", 
+      "Properties": {
+        "Code": {
+          "S3Bucket": {
+            "Ref": "BucketName"
+          }, 
+          "S3Key": "GetAllowedPlans/jars/GetAllowedPlans-0.0.1-SNAPSHOT.jar"
+        }, 
+        "MemorySize": "512",
```
Comment (akshayjoshi-hws) [inline GetAllowedPlans.template]: do we need 512 or is 256 enough

File: src/test/java/solutions/heavywater/helper/methods/HelperMethods.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+	public static HashMap<String, String> getConfigParams(String configString, String envVar, String stageVar){
+		HashMap<String, String> hm = new HashMap<String, String>();
+		try {
+			JSONObject configJson = new JSONObject(configString);
+			//System.out.println("ConfigJSON: \t"+configJson);
+			JSONObject envJson =  (JSONObject) configJson.get("environment");
+			//System.out.println("DEVJSON: \t"+envJson.get(envVar));
+			JSONObject specJson = (JSONObject) envJson.get(envVar);
+			JSONObject stageJson = (JSONObject) specJson.get(stageVar);
+			//System.out.println("SPECJSON: \t"+specJson);
```
Comment (akshayjoshi-hws) [inline src/test/java/solutions/heavywater/helper/methods/HelperMethods.java]: remove comments

File: src/test/resources/cucumber/acceptance/TestCases.feature [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+	
+Scenario: Positive test for service with mocking 
+	When I invoke the service with a "mockValidJson" payload 
+	Then I verify if the service returned a 200 
+	
+Scenario: Positive test for service 
+	When I invoke the service with a "validJson" payload 
+	Then I verify if the service returned a 200
+	
+#Scenario: Positive test for service
```
Comment (akshayjoshi-hws) [inline src/test/resources/cucumber/acceptance/TestCases.feature]: Is this scenario not needed

Comment (karansharmaufl) [reply src/test/resources/cucumber/acceptance/TestCases.feature]: Needed, Empower is currently down so this scenario can't be tested. Trying to ship this piece today.

File: GetAllowedPlans.template (hunk 2 above)
Comment (karansharmaufl) [reply GetAllowedPlans.template]: 256 should suffice, checked my test cases using 150MB per run. Reduced to 256

File: README.md (hunk 1 above)
Comment (karansharmaufl) [reply README.md]: My bad, fixed.

File: src/test/java/solutions/heavywater/helper/methods/HelperMethods.java (hunk 3 above)
Comment (karansharmaufl) [reply src/test/java/solutions/heavywater/helper/methods/HelperMethods.java]: done

File: src/test/resources/cucumber/acceptance/TestCases.feature (hunk 4 above)
Comment (akshayjoshi-hws) [reply src/test/resources/cucumber/acceptance/TestCases.feature]: Make sure a defect is created for this

**Pull request merged**